src/segment/writer.rs

```diff
@@ -153,7 +153,7 @@ impl SegmentWriter {
                     Ok((reply, connection)) => RetryResult::Success((reply, connection)),
                     Err(e) => {
                         info!("failed to setup append using rawclient due to {:?}", e);
-                        RetryResult::Retry(e)
+                        RetryResult::Fail(e)
```

Comment:
Why this change?

Reply:
reverted

made this change to see whether the recover speed will be improved